tests/TestCIT_KCI.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+import numpy as np
+
+import causallearn.utils.cit as cit
+
+class TestCIT_KCI(unittest.TestCase):
+    def test_Gaussian(self):
+        X = np.random.randn(300, 1)
+        X1 = np.random.randn(300, 1)
+        Y = np.concatenate((X, X), axis=1) + 0.5 * np.random.randn(300, 2)
+        Z = Y + 0.5 * np.random.randn(300, 2)
+
+        cit_CIT = cit.CIT(data=np.hstack((X,X1,Y,Z)), method='kci')
+
+        pvalue = cit_CIT.kci(0, 1, {})
+        self.assertTrue(pvalue > 0.01) # X and X1 are independent
+
+        pvalue = cit_CIT.kci(0, 3, {})
+        self.assertTrue(pvalue <= 0.01) # X and Z are dependent
+
+        pvalue = cit_CIT.kci(0, 3, {2})
```

Comment:
Should be e.g., `(0, 4, {2})` instead of `(0, 3, {2})` to test `X;Z|Y`, since Y and Z are two-dimensional.

Reply:
But overall this line is a very inspiring: KCI is capable of testing independence between multi-dimensional variables, e.g., `(0, [4, 5], [2, 3])` for `X;Z|Y`. However, the current `CIT` class only supports single-dimensional variables pair. What do you think? Should we also include this of KCI in CIT class?

Reply:
Yes, I think we should include this. We can do this later.